consoles/video_stream.pm

```diff
@@ -146,6 +147,7 @@ sub connect_remote_video ($self, $url) {
     }
 
     if ($url =~ m^ustreamer://^) {
+        die 'unsupported arch' unless ($Config{archname} =~ /^aarch64|x86_64/);
```

Comment:
I guess we need the host architecture. Was wondering since we never used this so far.

Reply:
well, don't we need the arch of whatever system does the `unpack`? which is the same process as hits this code, if I'm reading it right.